src/account/azext_account/__init__.py

```diff
@@ -19,8 +19,7 @@ def __init__(self, cli_ctx=None):
         account_custom = CliCommandType(
             operations_tmpl='azext_account.custom#{}',
             client_factory=cf_account)
-        super(SubscriptionClientCommandsLoader, self).__init__(cli_ctx=cli_ctx,
-                                                               custom_command_type=account_custom)
+        super().__init__(cli_ctx=cli_ctx, custom_command_type=account_custom)
```

Comment:
Why do we need to modify here?

Reply:
Syntax required in `azdev style`: `Consider using Python 3 style super() without arguments (super-with-arguments)`.